editor.planx.uk/src/pages/FlowEditor/components/Team/styles.ts

```diff
@@ -16,3 +17,14 @@ export const StyledTableRow = styled(TableRow)(({ theme }) => ({
     background: theme.palette.background.paper,
   },
 }));
+
+export const SettingsDialog = styled(Dialog)(({ theme }) => ({
```

Comment:
This works for now, but a future PR should refactor this into a global Editor style override in the `theme.ts` file